opcua/server/internal_server.py

```diff
@@ -332,7 +333,10 @@ def activate_session(self, params):
         self.state = SessionState.Activated
         id_token = params.UserIdentityToken
         if isinstance(id_token, ua.UserNameIdentityToken):
-            if self.iserver.allow_remote_admin and id_token.UserName in ("admin", "Admin"):
+            if self.iserver.user_token_manager:
+                if self.iserver.user_token_manager(id_token) == False:
+                    raise utils.ServiceError(ua.StatusCodes.BadUserAccessDenied)
+            elif self.iserver.allow_remote_admin and id_token.UserName in ("admin", "Admin"):
```

Comment:
should'nt we implement this default "admin" user using a default user_manager? instead of having a special case?

Reply:
That's a good idea.  I will implement that way.